src/scala/microsoft-spark-2.3.x/src/main/scala/org/apache/spark/api/dotnet/SerDe.scala

```diff
@@ -90,6 +95,13 @@ object SerDe {
     t
   }
 
+  def readRow(in: DataInputStream): Row = {
+    val len = readInt(in)
+    val rowArr = (0 until len).map(_ => readObject(in)).toArray
+    val rowValues = ListBuffer(rowArr: _ *)
+    Row.fromSeq(rowValues.toList)
```

Comment:
Does this work with something like the following ?
```scala
Row.fromSeq((0 until len).map(_ => readObject(in)).toArray)
```

Reply:
It gives an error: polymorphic expression cannot be instantiated to expected type;